@here/harp-geoutils/lib/coordinates/GeoBox.ts

```diff
@@ -198,6 +198,16 @@ export class GeoBox {
         return new GeoBox(this.southWest, this.northEast);
     }
 
+    /**
+     * Update the bounding box by considering a given point.
+     *
+     * @param point The point that may expand the bounding box.
+     */
+    expandByPoint(point: GeoCoordinates) {
```

Comment:
in harp-native this method is called growToContain. Please rename to be consistent

Reply:
I would say this should be renamed in a separate commit

Reply:
expandByPoint is used in THREE.js, so actually it wasn't chosen randomly.

Considering that we don't share code with HARP, I think the original name is better, but maybe that is just a personal opinion.